tests/operator/operator.go

```diff
@@ -3404,3 +3339,37 @@ func getDaemonsetImage(name string) string {
 
 	return matches[0][2]
 }
+
+func getComponentConfigPatchOption(toChange *v1.ComponentConfig, orig *v1.ComponentConfig, field string) patch.PatchOption {
+	path := "/spec/" + field
```

Comment:
nit:
WDYT about not limiting this to `/spec/` only?
Since this is anyhow handled by helper functions, you can provide the full path in them (e.g. `/spec/infra` in `patchKVInfra()`)